src/shared/lib/CivicUtils.ts

```diff
@@ -30,6 +30,8 @@ export function getCivicGenes(geneSymbols: Array<string>): Promise<ICivicGene> {
     let promises: Array<Promise<Array<ICivicGeneData>>> = [];
     let ids = '';
     geneSymbols.forEach(function(geneSymbol: string) {
+        //Encode "/" characters
+        geneSymbol = geneSymbol.replace(/[\/]/g,'%2F');
         // Check if we already have it in the cache
         if (civicGenes.hasOwnProperty(geneSymbol)) {
             return;
```

Comment:
another quibble: I noticed that we're building the ids string with concatenation.  would be better to build an array and then use the array.join method to build the coma delim'd string that is required by the client.